vizro-core/docs/index.md

```diff
@@ -2,64 +2,55 @@
 
 Vizro is a toolkit for creating modular data visualization applications.
 
+<div class="grid cards" markdown>
+
+-   :fontawesome-solid-forward-fast:{ .lg .middle } __New to Vizro?__
+
+    ---
+
+    Quick start tutorial to explore key features.
+
+    [:octicons-arrow-right-24: Get started](pages/tutorials/first-dashboard/)
+
+-   :fontawesome-solid-keyboard:{ .lg .middle } __Get hands-on__
+
+    ---
+
+    Step-by-step instructions on key Vizro features.
+
+    [:octicons-arrow-right-24: How-to guides](pages/user-guides/dashboard/)
+
+-   :fontawesome-solid-microscope:{ .lg .middle } __Get the detail__
+
+    ---
+
+    API reference for Vizro and our pydantic models.
+
+    [:octicons-arrow-right-24: API Reference](pages/API-reference/vizro/)
+
+-   :material-scale-balance:{ .lg .middle } __Vizro explained__
+
+    ---
+
+    Background information and answers to your "Why..?" questions.
+
+    [:octicons-arrow-right-24: Explanation](pages/explanation/why-vizro/)
+
+-   :fontawesome-solid-hands-holding-circle:{ .lg .middle } __Contribution guidelines__
+
+    ---
+
+    Find out more about contributing to Vizro.
+
+    [:octicons-arrow-right-24: Contribute](pages/development/contributing/)
```

Comment:
```suggestion
    [:octicons-arrow-right-24: Get started](pages/tutorials/first-dashboard.md)

-   :fontawesome-solid-keyboard:{ .lg .middle } __Get hands-on__

    ---

    Step-by-step instructions on key Vizro features.

    [:octicons-arrow-right-24: How-to guides](pages/user-guides/dashboard.md)

-   :fontawesome-solid-microscope:{ .lg .middle } __Get the detail__

    ---

    API reference for Vizro and our pydantic models.

    [:octicons-arrow-right-24: API Reference](pages/API-reference/vizro.md)

-   :material-scale-balance:{ .lg .middle } __Vizro explained__

    ---

    Background information and answers to your "Why..?" questions.

    [:octicons-arrow-right-24: Explanation](pages/explanation/why-vizro.md)

-   :fontawesome-solid-hands-holding-circle:{ .lg .middle } __Contribution guidelines__

    ---

    Find out more about contributing to Vizro.

    [:octicons-arrow-right-24: Contribute](pages/development/contributing.md)
```

Reply:
Hi @huong-li-nguyen I'm not sure of the changes in the docs links? Are you proposing changing them from `<prefix>/something/` to `<prefix/something.md`? because that's not working for me.

e.g. https://vizro--396.org.readthedocs.build/en/396/pages/tutorials/first-dashboard/ is OK but https://vizro--396.org.readthedocs.build/en/396/pages/tutorials/first-dashboard.md is not.

This is to do with the MkDocs default setting of `use_directory_settings` described https://www.mkdocs.org/user-guide/configuration/#use_directory_urls

Reply:
Ah! I've just spotted that something's changed in the RTD build. That'll be down to my setting up `--strict` for mkdocs... 😵‍💫 

I'll have to look into this. My comment above stands -- those URLs don't exist -- but I'll need to find out how to convince RTD of that!

Reply:
Fixed!